### ISSUE TITLE
Tá na hora de termos uma nova edição :)

### ISSUE THREAD

**Issue by erickmendonca** [pug-se/pythonday#1]
Just sayin' :shipit: :

**Comment by heat** [comment]
:+1:

**Comment by rodrigoamaral** [comment]
@erickmendonca tem um bom tempo que eu estava pra falar isso pra você e pro pessoal do GDG Aracaju. Ultimamente algumas pessoas têm entrado em contato comigo querendo saber quando teremos mais eventos presenciais sobre Python. Minha maior dificuldade geralmente é arrumar local. Tendo local resolvido é só juntar meia dúzia de pessoas e retomar os encontros do PUG-SE :smile: 

E aí? Vamos ou bora? @heat bora palestrar?

**Comment by erickmendonca** [comment]
Eu topo! Acho que a galera topa também :+1:

**Comment by davilima6** [comment]
:+1:

**Comment by heat** [comment]
Bora sim. O que vocês estavam pensando.  Queria q fosse algo com trilhas
- Python IoT
- Python aplicações
- Python científica

**Comment by erickmendonca** [comment]
Acho que até três trilhas fica bem bacana! Acho que uma delas tem de ser web pelo perfil dos profissionais mais comum.

**Comment by erickmendonca** [comment]
Galera, estou reunindo um pessoal de coisas comunidades para fazer um evento no segundo semestre. Sergipe Dev Day, topam?

**Comment by rodrigoamaral** [comment]
Claro! Vamos nessa :)

On Fri, Apr 1, 2016 at 8:51 PM Erick Mendonça notifications@github.com
wrote:

> Galera, estou reunindo um pessoal de coisas comunidades para fazer um
> evento no segundo semestre. Sergipe Dev Day, topam?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> https://github.com/pug-se/pythonday/issues/1#issuecomment-204610855
> 
> ## 

## 

Rodrigo Amaral
rodrigoamaral@gmail.com

**Comment by rodrigoamaral** [comment]
Pessoal, criei um repositório para facilitar a organização e divulgação desse e de outros eventos futuros https://github.com/pug-se/eventos

@erickmendonca assim que tiver mais detalhes, acho que podemos ir registrando por lá. O que acha?

**Comment by erickmendonca** [comment]
Opa! Além de outros eventos, né? A gente vai editando manualmente sempre que tiver um, é isso?

**Comment by rodrigoamaral** [comment]
Exato. A ideia também seria usar as [issues](https://github.com/pug-se/eventos/issues) e o [wiki](https://github.com/pug-se/eventos/wiki) para ajudar na organização. Já podemos começar com a participação Python no Sergipe Dev Day, a propósito :wink:
